content/docs/codebase-overview.md

```diff
@@ -66,7 +66,7 @@ The [fbjs repository](https://github.com/facebook/fbjs) exists because React sha
 After cloning the [React repository](https://github.com/facebook/react), you will see a few top-level folders in it:
 
 * [`src`](https://github.com/facebook/react/tree/master/src) is the source code of React. **If your change is related to the code, `src` is where you'll spend most of your time.**
-* [`docs`](https://github.com/facebook/react/tree/master/docs) is the React documentation website. When you change APIs, make sure to update the relevant Markdown files.
+* [`docs`](https://github.com/reactjs/reactjs.org/tree/master/content) is the React documentation website. When you change APIs, make sure to update the relevant Markdown files.
```

Comment:
Strictly speaking this line should be removed because it the docs is no longer part of the facebook/react repo.

Reply:
I think it's actually pretty important to keep this link here. 😄

Reply:
We should make a pass after repo re-organization and rewrite this.

Reply:
Definitely.

Reply:
Sorry I meant that it should be removed from this section and rephrased to suit the new repo structure. Definitely agree we should keep the link in this page!